drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -562,9 +570,12 @@ def _get_user_state(self):
 
     def _get_item_state(self):
         """
-        Returns the user item state.
+        Returns a copy of the user item state.
         Converts to a dict if data is stored in legacy tuple form.
         """
+
+        # IMPORTANT: this method should always return a COPY of self.item_state - it is called from get_user_state
+        # handler and manipulated there to hide correctness of items placed
```

Comment:
I think we should just move the code that removes `'correct'` entries in assessment mode into this method, as mentioned in a comment above. We can remove this comment then.